### DISCUSSION TITLE
Will it work with Alfred 5?

### DISCUSSION THREAD

**Discussion by jonasve** [mschmidtkorth/alfred-clickup-msk#41] [General]
Will your Plugin work with alfred 5 as well? Did anyone test it already?

**Comment by cdouglasnet** [comment]
@jonasve I know that this has been a little while...

There is now a fork of this project that is up and running (Python 3) and has fixed many of the issues here for those wanting to get ClickUp working with Alfred again. I tried working on it based on the work of @WreckItR but didn't get very far. Greg Flint with Four13 Digital is the one who got it working.
Here is his fork:

https://github.com/four13co/alfred-clickup-four13

Thanks @mschmidtkorth for all that you have done to get this project going and started.
Thanks @gregflint for working on this and getting things running on Newer Macs.
Thanks @WreckItR for working on this as well.

Hopefully this is helpful.

**Comment by mschmidtkorth** [comment]
Hi @jonasve @cdouglasnet I'm no longer actively working on this as I'm neither using Alfred nor ClickUp anymore.

Hi @cdouglasnet thank you for linking this fork, I've referenced it in the readme.
